css/style.css

```
@@ -210,6 +210,11 @@ img {
 	margin-left: 0;
 	padding-left: 0;
 }
+#search-results-header {
+	font-size: 2em;
+	font-weight: bold;
+	margin-top: 0.67em;
```

This is the actual `h1` styling from Bootstrap:

```suggestion
	margin: 0 0.67em 0;
```